src/Compilers/CSharp/Test/Symbol/Symbols/CheckedUserDefinedOperatorsTests.cs

```diff
@@ -2668,6 +2741,29 @@ static class C
                 );
         }
 
+        [Fact]
+        public void ConversionOperators_Supported_13()
+        {
+            var source1 =
+@"
+struct C 
+{
+    public static explicit operator checked int(C x) => 0;
+
+    public static explicit operator C(int x) => default;
+    public static explicit operator int(C? x) => default;
+    public static explicit operator long(C x) => default;
+}
+";
+            var compilation1 = CreateCompilation(source1, options: TestOptions.DebugDll, parseOptions: TestOptions.RegularPreview);
+
+            compilation1.VerifyDiagnostics(
+                // (4,45): error CS0216: The operator 'C.explicit operator checked int(C)' requires a matching operator 'explicit' to also be defined
```

Comment:
This error message seems weird? I would expect to be told which operator I need to actually define.

Reply:
> This error message seems weird? I would expect to be told which operator I need to actually define.

Could you give specific example given your understanding of the rules?

Reply:
I will add a PROTOTYPE comment to follow up on the wording.

Reply:
I think the thing throwing me is that the issue is with the lack of an _unmodified_ operator. I read this error message and go "I already have an explicit conversion operator, what else do you want from me?". As a member of the language design team I know what that means, but I don't expect it to be immediately obvious to most users.